packages/contracts/contracts/ChugSplashManager.sol

```diff
@@ -1,7 +1,10 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.9;
 
-import { Owned } from "@rari-capital/solmate/src/auth/Owned.sol";
+// import { Owned } from "@rari-capital/solmate/src/auth/Owned.sol";
```

Comment:
I say we just get rid of the import entirely. My preference is to stick to either OZ or solmate, since we're using a lot of OZ elsewhere let's do OZ.